statescript/executor.go

```diff
@@ -165,6 +167,19 @@ func execute(name string, timeout time.Duration) int {
 		return retCode(err)
 	}
 
+	bts, err := ioutil.ReadAll(stderr)
+	if err != nil {
+		log.Error(err)
+	}
+
+	if len(bts) > 0 {
+		if len(bts) > 10*1024*8 { // do not log more than 10KB at a time
```

Comment:
I think this should be `len(bts) > 10*1024`. `ioutil.ReadAll` is returning slice of byte (`[]byte`) which is `"byte is an alias for uint8 and is equivalent to uint8 in all ways. It is used, by convention, to distinguish byte values from 8-bit unsigned integer values."`.